src/drawing/backend_impl/svg.rs

```diff
@@ -247,24 +248,24 @@ impl<'a> DrawingBackend for SVGBackend<'a> {
         if color.alpha() == 0.0 {
             return Ok(());
         }
-        let context = svg::node::Text::new(text);
-        let layout = font.layout_box(text).map_err(DrawingErrorKind::FontError)?;
 
-        let trans = font.get_transform();
-        let offset = trans.offset(layout);
-        let x0 = pos.0 + offset.0;
-        let y0 = pos.1 + offset.1;
-
-        let max_x = (layout.1).0;
-        let (dx, anchor) = match style.alignment {
-            TextAlignment::Left => (0, "start"),
-            TextAlignment::Right => (max_x, "end"),
-            TextAlignment::Center => (max_x / 2, "middle"),
+        let (x0, y0) = pos;
+        let text_anchor = match style.pos.h_pos {
+            HPos::Left => "start",
+            HPos::Right => "end",
+            HPos::Center => "middle",
         };
+        let dominant_baseline = match style.pos.v_pos {
+            VPos::Top => "hanging",
```

Comment:
Just tried with `eog`, which is `librsvg` based SVG rendering, it seems it doesn't support this. So could we think about not supporting the vertical anchor point for now, because I think we should generate SVG that renders properly in most of the clients.

Reply:
Or at least, we don't rely on `dominant_baseline` for a while until this is widely supported.

Reply:
Thanks, I'll add the tests and try to find a solution.